### PR TITLE
Section 325 260 book 1

### PR DESCRIPTION
**Pull request by stevan04**
###
Implemented all section without any feature sections. For example "Text" and turn to.

###
Is needed

###
With the section yaml schema

Fixes https://github.com/keykey7/fabbwled/issues/201
Fixes https://github.com/keykey7/fabbwled/issues/223
Fixes https://github.com/keykey7/fabbwled/issues/447
Fixes https://github.com/keykey7/fabbwled/issues/448
Fixes https://github.com/keykey7/fabbwled/issues/449
Fixes https://github.com/keykey7/fabbwled/issues/450
Fixes https://github.com/keykey7/fabbwled/issues/451
Fixes https://github.com/keykey7/fabbwled/issues/452

### CODE REVIEW

**Review by dbabvwid — COMMENTED** [review]
Please link the Issues

**Review by adrsch04 — APPROVED** [review]
It looks good; maybe add newlines between the sections to improve readability.

**Comment by stevan04** [comment]
> Please link the Issues

did it in the pr. description

File: fabbwled-backend/src/main/resources/sections/325-360.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  content:
+    - text: |
+        You are too fast to get caught, and run off into the shadowy backstreets of Caran Baru.
+        Cursing your luck, you consider your next move.
+    - turnTo: 400
+- number: 352
+  content:
+    - text: |
+        You mutter an enchantment that is intended to give you the appearance of a rat man but you get some of the intricate syllables wrong,
+        and  take  on  the  appearance  of  a  particularly  wealthy merchant!
```
Comment (keykey7) [inline fabbwled-backend/src/main/resources/sections/325-360.yaml:50]: quite some double-spaces in here...

File: fabbwled-backend/src/main/resources/sections/325-360.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        - text: Stay hidden and follow when they leave
+          then:
+            - turnTo: 541
+- number: 360
+  content:
+    - text: |
+        You have no luck in tracking down the ghoul.
+        The next day, you hear a rumour doing the rounds in the taverns.
+        A ghoul was found and destroyed by an adventuring priestess of Tyrnai
+        –apparently, the temple of Nagil rewarded her handsomely.
```
Comment (keykey7) [inline fabbwled-backend/src/main/resources/sections/325-360.yaml:73]:
```suggestion
        – apparently, the temple of Nagil rewarded her handsomely. 
```

**Review by Nilstrieb — APPROVED** [review]

File: fabbwled-backend/src/main/resources/sections/325-360.yaml (hunk 1 above)
Comment (stevan04) [reply fabbwled-backend/src/main/resources/sections/325-360.yaml:50]: fixed.

**Review by kimberlymo — APPROVED** [review]
lgtm 👍

**Review by lucasmillanm — APPROVED** [review]
Seems okay

**Comment by stevan04** [comment]
suggestions were implemented 
pr got reviewed and merged

**Pull request merged**